packages/console/src/components/RequestDataError/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { AppearanceMode } from '@logto/schemas';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import RequestErrorDarkImage from '@/assets/images/request-error-dark.svg';
+import RequestErrorImage from '@/assets/images/request-error.svg';
+import type { RequestError } from '@/hooks/use-api';
+import { useTheme } from '@/hooks/use-theme';
+
+import Button from '../Button';
+import Card from '../Card';
+import * as styles from './index.module.scss';
+
+type Props = {
+  error: RequestError;
+  onRetry?: () => void;
+  className?: string;
+};
+
+const RequestDataError = ({ error, onRetry, className }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const theme = useTheme();
+  const errorMessage = error.body?.message ?? error.message;
+  const isNotFoundError = error.status === 404;
+
+  return (
+    <Card className={classNames(styles.error, className)}>
+      {theme === AppearanceMode.LightMode ? (
+        <RequestErrorImage className={styles.image} />
+      ) : (
+        <RequestErrorDarkImage className={styles.image} />
+      )}
```

Comment:
You can select the component first and then render it, in which way you can avoid defining the `className` twice. E.g.:
```ts
const Image = theme === AppearanceMode.LightMode ? RequestErrorImage : RequestErrorDarkImage;
...
<Image className={styles.image} />
```

Reply:
Updated.